.gitignore

```diff
@@ -1 +1,2 @@
-/vendor
\ No newline at end of file
+/vendor
+.idea
```

Comment:
we should have global git config to avoid these.

Reply:
meaning what exactly? That you'd prefer not to have this in the repo, and it should be deleted again?